include/tvm/relay/transform.h

```diff
@@ -191,11 +229,9 @@ class Sequential : public Pass {
    * \brief The constructor of `Sequential`.
    * \param passes The passes to apply.
    * \param pass_info The pass metadata.
-   * \param disabled The passes that will not be applied.
    */
   TVM_DLL Sequential(tvm::Array<Pass> passes,
-                     PassInfo pass_info,
-                     tvm::Array<tvm::Expr> disabled);
+                     PassInfo pass_info);
```

Comment:
we want to also enable TVM_DLL Sequential(List<Pass>, name="Sequential"); think about how to minimize user's code

Reply:
This looks from the old commit. `disabled` should have been moved.